java/code/src/com/suse/manager/webui/utils/gson/ChannelsJson.java

```diff
@@ -51,13 +53,16 @@
          * @param customIn custom channel flag
          * @param subscribableIn subscribable flag
          * @param recommendedIn the channel is recommended by its parent channel
+         * @param wasSelectedOnOldBaseChannelIn the channel was selected on the previous baseChannel
```

Comment:
Why not something like `isCompatibleWithPreviousSelection` or `isCompatibleWithOldSelection`?

I know, as usual, naming things is something really hard, I already like your naming, but I am looking for something that explains the *feature intention* itself. Reading `wasSelectedOnOldBaseChannelIn` could mislead the fact that exactly the same child channel was selected already before, and this is not the case since we are pre-selecting the **correspondent** child channel under the new base channel, which is different from the previous one (`channel id` is different for instance). It's just a hint to the future developer who will need to take a look here. Sorry for being picky :wink:

Reply:
yeah, isCompatibleWithPreviousSelection seems better, I will rename it